.github/workflows/versioned-coverage.yml

```diff
@@ -7,6 +7,7 @@
 name: Gather Versioned Test Coverage
 
 on:
+  workflow_dispatch:
```

Comment:
You know what I was going to mention this but didn't want to make any more requests after something gets merged 😄 . good thinking though

Reply:
please wait to merge until we get 9.5.0 out though